src/test/resources/config.json

```diff
@@ -17,6 +17,7 @@
       "browser.download.folderList": 2
     }
   },
+  "exceptionsWaitTimeOut": "5",
```

Comment:
I think this is badly named.  I think we need to tell the user what the unit of time is.  Maybe something like `exceptionWaitTimeoutInSeconds`.

Additionally, I think the JSON is now poorly formed because we have a two values that are related but not in the same object.  I think we need an object like:

`
exceptions: {
    waitTimeoutInSeconds: "5",
    exceptionsToHandleOnTolerantActions: {}
}
`

Reply:
Created new class to represent Tolerant exceptions and wait time as mentioned above 
and created new method in the WebDriver config if user defined the tolerantActionWaitTimeoutInSeconds then it will read from config other wise it returns WebDriver waitTimeout